docs/querying/api.md

```diff
@@ -1393,3 +1393,54 @@ Enable the OTLP receiver by setting
 endpoint is `/api/v1/otlp/v1/metrics`.
 
 *New in v2.47*
+
+## Notifications
+
+The following endpoints provide information about active notifications in
```

Comment:
Since notifications is an overloaded term (there's a whole metrics namespace `prometheus_notifications_...` about alert notifications), let's clarify here what these notifications are about. How about something like:

"The following endpoints provide information about active status notifications concerning the Prometheus server itself."